js/components/controlBar.js

```diff
@@ -879,33 +876,38 @@ var ControlBar = createReactClass({
         alignment = CONSTANTS.TOOLTIP_ALIGNMENT.LEFT;
       }
       tooltipAlignments[collapsedControlBarItems[k].name] = alignment;
-      finalControlBarItems.push(controlItemTemplates[collapsedControlBarItems[k].name]);
+      var item = controlItemTemplates[collapsedControlBarItems[k].name];
+      if (this.props.equalSpacing) {
+        item = (
+          <div className="oo-flex-row" key={collapsedControlBarItems[k].name}>
+            {item}
+          </div>
+        );
+      }
+      finalControlBarItems.push(item);
     }
+
     return finalControlBarItems;
   },
 
   render: function() {
     var controlBarClass = ClassNames({
       'oo-control-bar': true,
-      'oo-control-bar-hidden': !this.props.controlBarVisible
+      'oo-control-bar-hidden': !this.props.controlBarVisible,
+      'oo-control-bar-video': !this.props.simpleControlBar
     });
 
     var controlBarItems = this.populateControlBar();
-    var controlBarStyle = {
```

Comment:
Does removing this have any effect on the regular player mode?

Reply:
I forgot to add a comment about this. The height for the control bar is defined in our css. I wanted QA to see if removing this affected anything, since there were two sources of truth. Nice catch though!

Reply:
I'm going to add this back in. I did not notice that it's taking the height from the skin config